__tests__/app.test.js

```diff
@@ -99,6 +99,55 @@ describe('app', () => {
                         expect(reviews).toEqual([])
                     })
                 })
+                test('status 200: responds with reviews sorted by given column - votes', () => {
```

Comment:
Really picky, but i'd take the specification of 'votes' out of the description here. It reads as though this is testing for votes only.